From 7ea14bbaf6b0e923496cb5e50dfcd5068c7858c5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bj=C3=B6rn=20Feber?= <bfeber@protonmail.com>
Date: Sun, 30 Jun 2019 17:50:38 +0200
Subject: [colord-kde] Use more appropriate icon, because the activities icon
 got changed

Summary:
Use a more appopriate icon, because the activities icon got changed in D22155.
{F6948366}

Test Plan: Open the Color Corrections KCM.

Reviewers: #plasma, #vdg, ndavis

Reviewed By: #vdg, ndavis

Subscribers: #vdg, #plasma

Differential Revision: https://phabricator.kde.org/D22171
---
 colord-kcm/ColordKCM.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/colord-kcm/ColordKCM.cpp b/colord-kcm/ColordKCM.cpp
index f14b4a9..fffbf54 100644
--- a/colord-kcm/ColordKCM.cpp
+++ b/colord-kcm/ColordKCM.cpp
@@ -158,7 +158,7 @@ ColordKCM::ColordKCM(QWidget *parent, const QVariantList &args) :
     connect(watcher, &QDBusServiceWatcher::serviceOwnerChanged,
             ui->profile, &Description::serviceOwnerChanged);
 
-    ui->devicesTb->setIcon(QIcon::fromTheme(QStringLiteral("preferences-activities")));
+    ui->devicesTb->setIcon(QIcon::fromTheme(QStringLiteral("computer")));
     ui->profilesTb->setIcon(QIcon::fromTheme(QStringLiteral("application-vnd.iccprofile")));
 
     auto signalMapper = new QSignalMapper(this);
-- 
cgit v1.1
